modules/nf-core/bcftools/pluginimputeinfo/meta.yml

```diff
@@ -4,15 +4,28 @@ description: Adds imputation information metrics to the INFO field based on sele
 keywords:
   - impute-info
   - bcftools
+  - imputation
+  - metrics
+  - tags
   - VCF
 tools:
   - "bcftools":
+      description: "BCFtools is a set of utilities that manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF. All commands work transparently with both VCFs and BCFs, both uncompressed and BGZF-compressed.  Most commands accept VCF, bgzipped VCF and BCF with filetype detected automatically even when streaming from a pipe. Indexed VCF and BCF will work in all situations. Un-indexed VCF and BCF and streams will work in most, but not all situations."
+      homepage: "http://samtools.github.io/bcftools/bcftools.html"
+      documentation: "https://samtools.github.io/bcftools/bcftools.html#plugin"
+      tool_dev_url: "https://github.com/samtools/bcftools"
+      doi: "10.1093/bioinformatics/btp352"
+      licence: ["MIT"]
+      args_id: "$args"
```

Comment:
I added a separate "bcftools plugin impute-info" entry into tools to account for `args2` but wasn't sure if there is a better way to document this?

Reply:
This is fine! I would just maybe change or reduce and clarify the descirption for the plugin bit to say what bcftool plugins can do etc.